Source/WebCore/html/canvas/WebGLRenderingContextBase.h

```diff
@@ -183,6 +184,9 @@ class WebGLRenderingContextBase : public GraphicsContextGL::Client, public GPUBa
     int drawingBufferWidth() const;
     int drawingBufferHeight() const;
 
+    PredefinedColorSpace drawingBufferColorSpace() const;
+    void setDrawingBufferColorSpace(PredefinedColorSpace colorSpace);
```

Comment:
Nit: in WebKit we tend to omit argument names when they don't add anything over the type. So for this one I'd drop the `colorSpace`.